pkg/daemon/ceph/osd/health_test.go

```diff
@@ -84,9 +84,9 @@ func TestOSDStatus(t *testing.T) {
 	// After creating an OSD, the dump has to be the 4th mocked cmd
 	assert.Equal(t, 4, execCount)
 	// Only one OSD proc was mocked
-	assert.Equal(t, 1, len(agent.osdProc))
-	// OSD monitoring should start tracking an osd with Down status
-	assert.Equal(t, 1, len(osdMon.lastStatus))
+	//assert.Equal(t, 1, len(agent.osdProc))
+	// FIX: OSD monitoring should start tracking an osd with Down status
```

Comment:
The "finalize whether monitor OSD health in prepare pod or operator (from @travisn and @galexrt )" covers this, right?